main.tex

```diff
@@ -1259,13 +1259,13 @@ \subsubsection{Stability Pool Staking
   \(\varphi\) is the sequence of Stability Pool saturation values of
   each iAsset
 \item
-  \(\omega\) is the sequence of market caps of each iAsset
+  \(\omega\) is the sequence of market cap values of each iAsset
 \item
   \(t\) is the sequence of total number of Stability Pool stakers, who
-  have staked for at least 24 hours, of each iAsset
+  have staked for at least 24 hours, for all iAssets
```

Comment:
"of each iAsset" is more consistent and I believe fits the context. "for all iAssets" changes the meaning somewhat and makes it a bit more ambiguous imo